pyriemann_qiskit/classification.py

```diff
@@ -198,10 +200,7 @@ def _init_algo(self, n_features):
 
     def _train(self, X, y):
         self._log("Training...")
-        if self.quantum:
-            self._classifier.train(X, y, self._quantum_instance)
-        else:
-            self._classifier.train(X, y)
+        self._classifier.fit(X, y)
```

Comment:
QSVC now inherint from SVC, so they have the same `fit`, `score` and `predict` methods.